webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -318,8 +318,7 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     if (!isLoadingSettings && settings.sortString) {
       setSorts(parseSortString(settings.sortString));
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@EmilyBonar its working without it, but in case im missing something,  im trying to understand why this is added. do you have more context?

Reply:
I don't remember, but I assume the goal was to only read the sortString from settings once on page load.